File: MainProject/app/Model/element.ts


export class Element {
    Id: number;
    StepId: number;
    Data: string;
    BlockType: number;
    Number: number;

    constructor(id: number, stepId: number, blocT: number,num:number) {
        this.Id = id;
        this.StepId = stepId;
        this.BlockType = blocT;
        this.Number = num;
       
    }
}

File: MainProject/Models/Element.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace MainProject.Models
{
    public class Element
    {
        public int Id { get; set; }
        public int BlockType { get; set; }
        public int Number { get; set; }
        public string Data { get; set; }
        public int? StepId { get; set; }
        public Step Step { get; set; }

    }
}